src/infrastructure/data-access/typeorm/user.entity.ts

```diff
@@ -17,7 +18,7 @@ export class UserModel {
   @Column({
     nullable: false,
```

Comment:
```suggestion
    nullable: false,
```
nullable:false or true?